src/front/app/components/challengeTopics/ChallengeTopics.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ChallengeTopic from './ChallengeTopic';
+
+let ChallengeTopics = ({challengeTopics = [], getChallengeTopics}) => {
+    if(!challengeTopics.length) {
+        getChallengeTopics();
+    }
+
+    return (
+        <ul>
+            {challengeTopics.map((challengeTopic, index) => {
+                return (
+                    < ChallengeTopic key={index} challengeTopic={challengeTopic}/>
+                );
+            })}
+        </ul>
+    );
+};
+
+const mapStateToProps = (state) => {
+    return {
+        challengeTopics: state.app.challengeTopics
+    };
+};
+
+const mapDispatchToProps = (dispatch) => {
+    return {
+        getChallengeTopics: () => {
+            dispatch({type: 'GET_CHALLENGE_TOPICS'})
```

Comment:
please avoid usage of raw literals, better to move in 'GET_CHALLENGE_TOPICS' into module
eg: 
```js
// challangeTopicsActions.js
export const GET_CHALLENGE_TOPICS = 'GET_CHALLENGE_TOPICS';
```